app/assets/javascripts/search.js

```diff
@@ -13,7 +13,11 @@ function search() {
         var searchTitle = title.indexOf(searchInput) >= 0;
         var searchBody  = body.indexOf(searchInput) >= 0;
 
-        (searchTitle || searchBody) ? $(this).show() : $(this).hide();
```

Comment:
@rrgayhart I personally prefer the ternary for the basic if/else statements, I think it looks cleaner, but what is the general consensus among the community?